shard.yml

```diff
@@ -1,9 +1,14 @@
 name: wordsmith
-version: 0.3.0
+version: 0.3.2
```

Comment:
You can leave this off. I'll update it when I do a release.

Reply:
Done in 7b20234